spec/ParseLiveQuery.spec.js

```diff
@@ -393,6 +393,49 @@ describe('ParseLiveQuery', function () {
     await object.save();
   });
 
+  xit('can handle live query with fields - enable upon JS SDK support', async () => {
```

Comment:
Could you explain this xit a bit more?

Reply:
Once the JS SDK supports the `.listen` method and is released, the test should be enabled. So it's:

1. Create feature in Parse Server
2. Merge commit into JS SDK for integration tests (dependabot)
3. Update JS SDK for `ParseQuery.listen` + unit/integration tests
4. Update JS SDK dependancy in Parse Server 
5. Enable the `.listen` test in Parse Server

Reply:
So this PR needs to be merged as step 1?

Reply:
Yep!